---
# Display name
#name: Lu Zhang
#name_zh: å¼ éœ²

# Status emoji
status:
  icon: ðŸŽ¨

# Is this the primary user of the site?
superuser: true

# Highlight the author in author lists? (true/false)
highlight_name: true

# Role/position/tagline
role: Lu Zhang

# Organizations/Affiliations to display in Biography blox
organizations:
  - name: Stanford University
    url: https://profiles.stanford.edu/351436

# Social network links
# Need to use another icon? Simply download the SVG icon to your `assets/media/icons/` folder.
profiles:
  - icon: at-symbol
    url: 'mailto:luzhang0828@gmail.com'
    label: E-mail Me
  - icon: brands/x
    url: https://x.com/LuZhang17593753
  - icon: brands/github
    url: https://github.com/wbszhu

interests:
  - Three-dimensional genomics
  - Single-cell multi-omics
  - Spatial omics
  - Deep Learning in Genomics

education:
  - area: PhD in Animal Genetics, Breeding, and Reproduction
    institution: Huazhong Agricultural University
    date_start: 2020-09-01
    date_end: 2024-06-30
  - area: Visiting PhD in Animal Genetics, Breeding, and Reproduction
    institution: Center for Quantitative Genetics and Genomics, Aarhus University
    date_start: 2023-11-01
    date_end: 2024-04-20
  - area: Exchange Student in Animal Science
    institution: South China Agricultural University
    date_start: 2014-09-01
    date_end: 2016-06-30
  - area: BSc in Animal Science
    institution: ShiHeZi University
    date_start: 2013-09-01
    date_end: 2017-06-30

# Skills
# Add your own SVG icons to `assets/media/icons/`
skills:
  - name: Technical Skills
    items:
      - name: Python
        description: ''
        percent: 80
        icon: code-bracket
      - name: Data Science
        description: ''
        percent: 100
        icon: chart-bar
      - name: SQL
        description: ''
        percent: 40
        icon: circle-stack
  - name: Hobbies
    color: '#eeac02'
    color_border: '#f0bf23'
    items:
      - name: Hiking
        description: ''
        percent: 60
        icon: person-simple-walk
      - name: Painting
        description: ''
        percent: 100
        icon: brush
      - name: Fishing
        description: ''
        percent: 80
        icon: Fish

languages:
  - name: English
    percent: 75
  - name: Chinese
    percent: 100

# Awards.
#   Add/remove as many awards below as you like.
#   Only `title`, `awarder`, and `date` are required.
#   Begin multi-line `summary` with YAML's `|` or `|2-` multi-line prefix and indent 2 spaces below.
awards:
  - title: Hi-Tag a simple and efficient method for identifying protein-mediated long-range chromatin interactions with low cell numbers
    url: https://www.sciengine.com/SCLS/doi/10.1007/s11427-023-2441-0;JSESSIONID=621f95e0-29ae-4c98-8b27-5aabb650fd6b
    date: '2024-01-23'
    awarder: Coursera
    icon: coursera
    summary: |
      Building on the foundations of BL-HiChIP, in 2023, our team developed Hi-Tag, a new chromatin spatial conformation capture technology tailored for low-cell-number samples. This technology simplifies the experimental process and optimizes the data processing method. It only requires 100,000 cells and allows for the identification of high-confidence chromatin loops mediated by specific histones or transcription factors without the need for additional ChIP-Seq/CUT&Tag data. The Hi-Tag technology was presented through a poster at the 10th International Conference on 3D Genomics in Hangzhou, Zhejiang, where it received appreciation from many researchers.
  - title: Enhancer-promoter interaction maps provide insights into skeletal muscle-related traits in pig genome
    url: https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-022-01322-2
    date: '2022-06-09'
    awarder: edX
    icon: edx
    summary: |
      In 2022, our research team developed the BL-HiChIP technique. By applying this technology in conjunction with GRID-seq and GWAS data, we successfully identified functional genetic variants associated with key agricultural traits, such as KLF6 (related to days to 100 kg), MXRA8 (related to lean meat percentage), and TAF11 (related to loin muscle depth). 
  - title: Identification and Conservation Analysis of Cis-Regulatory Elements in Pig Liver
    url: https://pubmed.ncbi.nlm.nih.gov/31067820/
    date: '2019-05-07'
    awarder: datacamp
    icon: datacamp
    summary: |
      The liver is crucial in metabolism and pig production, yet the functional annotation of pig liver's noncoding regions is underexplored. We identified 102,373 cis-regulatory elements in pig liver, including enhancers, promoters, super-enhancers, and broad H3K4me3 domains, and highlighted 26 core transcription regulatory factors. Our findings showed similarities between pig, human, and cattle cis-regulatory elements. Although only ~30% of pig liver enhancers are functionally conserved in human liver tissue, ~78% of pig liver enhancer orthologues can function in other human tissues. We observed that genes associated with super-enhancers have a higher conservation rate, with ~54% of core regulatory factors being consistent across the liver of these species. This study provides a valuable resource for future gene regulatory research in pigs and enhances our understanding of cis-regulatory element functions across mammals.
---

## About Me
I am a postdoctoral research scholar in Dr.Anna Gloyn's Translational Genomics of Diabetes Lab. During my master's and doctoral studies, I focused on epigenomics and single-cell multi-omics analysis, with a particular emphasis on 3D genomics. My research included developing Hi-Tag, a chromatin conformation capture technique designed specifically for small cell samples. This method provides valuable insights into the organization of chromatin within the cell.

I have built strong expertise in integrating various types of biological data, including RNA-seq, ATAC-seq, chromatin interaction data, and single-cell data. I have also contributed as a co-author to several research projects, including studies that utilized genome-wide association studies (GWAS) and GTEx data to link multi-omics data with functional genomics. These experiences have allowed me to develop a deep understanding of how to integrate diverse genomic data to address complex biological questions.

Currently, I am applying my research skills to explore questions related to diabetes.
